index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Survey form</title>
+
+  <link rel="stylesheet" href="mystyle.css">
+
+  <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ] Great job :+1: When you give a code review, make your suggestion clear and supportive with some reference for more clarification and be human-friendly with better emojis in the inline comments. Kindly use the example below as a reference to fix the issue mentioned above.

***For example***, "It would be better to use the meta tag containing charset attribute must be the first child sequence in the head tag it's an HTML standard to display HTML code correctly, and also good practice :cool: . For further details [check this.](https://www.w3schools.com/html/html_charset.asp) :book: "